packages/blade/src/components/VisuallyHidden/__tests__/VisuallyHidden.native.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Text } from 'react-native';
+import renderWithTheme from '../../../_helpers/testing/renderWithTheme.native';
+import VisuallyHidden from '../VisuallyHidden';
+
+describe('<VisuallyHidden />', () => {
+  // TODO: Enable css testing
+  it('should render VisuallyHidden content', () => {
+    const displayText = 'Text only visible to screen readers';
+    const { toJSON, getByText } = renderWithTheme(
+      <VisuallyHidden>
+        <Text>{displayText}</Text>
+      </VisuallyHidden>,
+    );
+    expect(getByText(displayText)).toBeTruthy();
```

Comment:
how is this returning truthy? or is that expected?

Reply:
yeah correct, the test is partially implemented then because what we need to also make sure is that it's visually hidden. can we add an assertion for that as well? maybe [this might help](https://github.com/testing-library/jest-dom#tobevisible)?

`expect(getByText(displayText)).not.toBeVisible();`

Reply:
while snapshot does that but it's just implying the CSS not the expectation